packages/manager/apps/web/client/app/hosting/multisite/git-association/hosting-multisite-git-association.controller.js

```diff
@@ -0,0 +1,37 @@
+import {
+  REPOSITORY_PLACEHOLDER,
+  EXAMPLE_BRANCHES_NAMES,
+  EXAMPLE_HTTPS_REPOSITORY_URL,
+  EXAMPLE_SSH_REPOSITORY_URL,
+  GITHUB_VCS,
+} from './hosting-multisite-git-association.constants';
+
+export default class HostingMultisiteGitAssociationController {
+  constructor(HostingMultisiteGitAssociationService) {
+    this.repositoryPlaceholder = REPOSITORY_PLACEHOLDER;
+    this.EXAMPLE_BRANCHES_NAMES = EXAMPLE_BRANCHES_NAMES;
+    this.EXAMPLE_HTTPS_REPOSITORY_URL = EXAMPLE_HTTPS_REPOSITORY_URL;
+    this.GITHUB_VCS = GITHUB_VCS;
+    this.EXAMPLE_SSH_REPOSITORY_URL = EXAMPLE_SSH_REPOSITORY_URL;
+    this.HostingMultisiteGitAssociationService = HostingMultisiteGitAssociationService;
+    this.model = {
+      repositoryUrl: null,
+      branchName: null,
+      webhookUrl: '',
```

Comment:
Why `null ` or `''` ? Not possible to initialize with the same value to be consistent

Reply:
it is set to '' because otherwise the oui clipboard will have issue with null